editor/editor_file_system.cpp

```diff
@@ -1865,13 +1865,14 @@ void EditorFileSystem::_find_group_files(EditorFileSystemDirectory *efd, Map<Str
 }
 
 void EditorFileSystem::reimport_files(const Vector<String> &p_files) {
-	{ //check that .import folder exists
+	{
+		// Ensure that IMPORTED_FILES_PATH exists (res://.godot/imported).
```

Comment:
If we hardcode the value in a comment, we lose part of the point of making it a constant in the first place :P

Reply:
Eh, I think it was fine, but I removed it from the comment.